.vscode/launch.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Need to remove this too along with the ignore

Reply:
This file was meant to be tracked bt git. See:

https://github.com/github/gitignore/blob/main/Global/VisualStudioCode.gitignore

It allows anyone who is using the devcontainer to easily debug it by simply pressing F5.